doc/cascadia/profiles.schema.json

```diff
@@ -378,6 +378,16 @@
           "minimum": 0,
           "type": "number"
         },
+        "antialiasingMode": {
+          "default": "grayscale",
+          "description": "Controls how text is antialiased in the renderer. Possible values are \"grayscale\", \"cleartype\" and \"aliased\". Note that changing this setting will require starting a new terminal instance.",
```

Comment:
Explain the default maybe? Grayscale is faster than cleartype but doesn't look as jagged as aliased. It's a compromise.